app/server.ts

```diff
@@ -20,11 +21,24 @@ const isDevelopmentEnv = process.env.NODE_ENV === 'dev';
 i18n.configure({
   locales: ['en', 'no', 'nl'],
   defaultLocale: LANGUAGE,
+  updateFiles: false,
   directory: `${__dirname}/locales`
 });
 
 app.use(i18n.init);
 
+app.use((req, res, next) => {
+  // @ts-ignore
```

Comment:
Just add eslint-ignore for these errors

Reply:
```
/**
     * Translate the given phrase using locale configuration
     * @param phraseOrOptions - The phrase to translate or options for translation
     * @returns The translated phrase
     */
    function __(phraseOrOptions: string | TranslateOptions, ...replace: string[]): string;
    /**
     * Translate the given phrase using locale configuration
     * @param phraseOrOptions - The phrase to translate or options for translation
     * @param replacements - An object containing replacements
     * @returns The translated phrase
     */
    function __(phraseOrOptions: string | TranslateOptions, replacements: Replacements): string;
```

These are the signatures of `__`, I don't know how to translate that into one function as we have in our middleware. The easiest thing to do is to just ts-ignore and eslint-ignore or way out of the problem 😆

Reply:
Something like this:
```typescript
app.use((req, res, next) => {
  const translate = (
    text: string,
    ...rest: string[] | [Replacements]
  ): string => {
    const modifiedText = text.replace(/[\s\n\t]+/g, ' ').trim();
    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
    // @ts-ignore
    return i18n.__.apply(req, [modifiedText, rest]);
  };
  res.locals.__ = translate;
  res.__ = translate;
  next();
});
```

Reply:
It said `Cannot find name 'Replacements'.`, just added `eslint-disable-next-line @typescript-eslint/ban-ts-ignore` now.

Reply:
Ah sorry you need the following import:
```import i18n, { Replacements } from 'i18n';```
But whatever, as long as it works :)